.circleci/install_testing_tools.sh

```diff
@@ -19,6 +21,8 @@ sudo mkdir -p "/usr/local/chart-testing-v$CHART_TESTING_VERSION"
 sudo tar -xzf "chart-testing_${CHART_TESTING_VERSION}_linux_amd64.tar.gz" -C "/usr/local/chart-testing-v$CHART_TESTING_VERSION"
 sudo ln -s "/usr/local/chart-testing-v$CHART_TESTING_VERSION/ct" /usr/local/bin/ct
 rm -f "chart-testing_${CHART_TESTING_VERSION}_linux_amd64.tar.gz"
+pip3 install "yamllint==${YAMLLINT_VERSION}"
```

Comment:
Updated

Reply:
I wasn't able to get `python3 -m pip install` to work (I believe because it installs it locally instead of globally) and I'm not super interested in debugging python right now. The chart-testing action uses pip3 as well: https://github.com/helm/chart-testing-action/blob/86b540ddcecb3cc009fa2bc0f44fa5b33e9751a2/ct.sh#L119-L122